apps/web/src/components/Checkboxes.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+import { FormFieldProps } from './Radio';
+
+function Checkboxes(props: FormFieldProps) {
+  const { label, name, options, ...rest } = props;
+  return (
+    <div>
+      <label>{label}</label>
+      <Field name={name}>
```

Comment:
I think this approach is unnecessary. Why do we need to wrap checkboxes with Field component? I rather do Field type="checkbox" approach